packages/source/Rakefile

```diff
@@ -0,0 +1,24 @@
+require_relative "../mroonga-package-task"
+
+class SourceMroongaPackageTask < MroongaPackageTask
+
+  def initialize
+    super("")
+  end
+
+  def built_package_url(target_namespace, target)
+    url = "https://github.com/mroonga/mroonga/releases/download/v#{@version}/"
+    url << "#{target}"
+    url
+  end
```

Comment:
How about generalize `MroongaPackageTask#built_package_url` and use common code?

```diff
diff --git a/packages/mroonga-package-task.rb b/packages/mroonga-package-task.rb
index e435a320..1b599978 100644
--- a/packages/mroonga-package-task.rb
+++ b/packages/mroonga-package-task.rb
@@ -210,10 +214,12 @@ class MroongaPackageTask < PackagesGroongaOrgPackageTask
     "ghcr.io/mroonga/package-#{super}"
   end
 
+  def download_url(file_name)
+    "https://github.com/mroonga/mroonga/releases/download/v#{@version}/#{file_name}"
+  end
+
   def built_package_url(target_namespace, target)
-    url = "https://github.com/mroonga/mroonga/releases/download/v#{@version}/"
-    url << "#{@package}-#{target}.tar.gz"
-    url
+    download_url("#{@package}-#{target}.tar.gz")
   end
 
   def built_package_n_split_components
```